### DISCUSSION TITLE
对地图表示严重不满！

### DISCUSSION THREAD

**Discussion by ipdear** [WeNeedHome/SummaryOfLoanSuspension#604] [General]
地图缺少藏南地区，缺少阿克赛钦区域。

**Comment by HunterTOne** [comment]
你去画一个

**Comment by ldmid666** [comment]
有必要争执这些吗，都是开源免费的东西，国内自己争执。这不是在国内

**Comment by ethenlong** [comment]
还缺外蒙古，海参崴？

**Comment by masami1912** [comment]
> 地图缺少藏南地区，缺少阿克赛钦区域。
> 
> 

我要花一辈子积蓄买的房子都没了，你给我说这个

**Reply by monkery** [reply]
知道啥叫“统一战线”不？大家的目标，应该是尽量团结一切可以团结的力量，而不是因为有人夹带私活而失去应有的基本盘。

**Comment by user-hook-name** [comment]
是台湾省不是台湾
政治嗅觉+1

**Reply by hktk1024** [reply]
可惜拿身份证去不了这个省😄

**Reply by hdp521** [reply]
名字都不是一个，你说这个？
3大运营商都知道是国际长途，你说这个？

**Reply by lance111l** [reply]
特区而已 还是中国的  当时进深圳也要申请

**Comment by liuwenwu520** [comment]
用的国外的地图当然有这样的情况，用国内的地图在这个敏感的时间容易喝茶呀

**Comment by Mmmmeruem** [comment]
这就是世人眼里的中国，别什么东西都看不得

**Comment by xiaoheikejj** [comment]
而且台湾这两个字太大了，应该和国内省份字体一样大小

**Comment by yalo-ddaa** [comment]
修改不了。建议你自己画一个，然后提交PR合并过来不就行了？有这份心就多做点实事。

**Comment by trunklater** [comment]
建议先下了那个地图吧，那个谷歌地图不完整。不要给ZY留理由，都是理性维权。

**Comment by tencentbook** [comment]
我估计是没法用国内地图吧，在国内讨论烂尾楼的链接都被封了，所以不得已用国外的地图。如果国内可以光明正大的上这种项目进行讨论也不会有地图的问题。

**Comment by killdayu** [comment]
已经删除了

**Comment by MarkShawn2020** [comment]
地图是我画的，我来解释一下：
1. 我自己对地图和地域没有个人偏好，由于技术框架原因，我也分别使用过高德、百度、谷歌的地图，在融合上不太理想，我时间也有限，本着使用可能技术水准更高的态度最后选用了谷歌地图
2. 初版画的地图，由于没有配置语言选项，所以导致了台湾变成了国家（国家字体会更大，且是白色，而省份是灰色），这个我确实没有注意，后来其他开发者把它删除了
3. 这里再展开解释一下，谷歌地图确实很鸡贼，如果语言是中文，就会显示为台湾省；如果不是中文，则是台湾国（如果仔细调整国家的样式，会发现区别），鉴于此，目前使用中文，就避免了争议问题，但本人对谷歌地图的做法不置评论，我只是用地图的，尽量呈现好的效果而已
4. 最后关于地图不完整的问题，这个我个人觉得不要太操心，其实没有是好事，因为是尽量把所有有停贷的城市都给显示出来，没有停贷的那些城市其实是默认可以出地图外的（比如东北的一部分也丢了），如果单纯为了地图完整，那就要把地图缩小，显示效果上不是很佳
以上。

**Reply by lance111l** [reply]
谷歌不收规矩 已被赶出中国了
